src/sentry/utils/http.py

```diff
@@ -120,15 +129,6 @@ def is_valid_origin(origin, project=None, allowed=None):
     if origin == "null":
         return False
 
-    if isinstance(origin, bytes):
```

Comment:
I am not convinced this is safe. What if this function is called from untyped code?